### PR TITLE
Ch-setup-sass-processing-181414556

### PR DESCRIPTION
**Pull request by iyaremye-faustin**
### 1. Name of task
-Setup SASS processing on the client
### 2. Pivot Tracker ID
- #181414556
### 3. Branch Name
- ch-setup-sass-processing-#181414556
### 4. How To Test The Functionality
-   On the root folder of your project do **yarn install** and start the app with **yarn run dev**
- Open the project in editor of choice and inside **src/sass/DemoSassCss.scss** change nav-demo class attributes values or add new ones to see how the style of a div on the page changes from a compiled sass file into css. 
- You have to run **yarn run sass** to work and keeping it running for watching all the changes

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
+import '../css/App.css';
```
Comment (SergeNm) [inline src/components/app.js:4]: Try to style the dummy component with sass

**Review by SergeNm — CHANGES_REQUESTED** [review]

File: src/components/app.js (hunk 1 above)
Comment (rukundo-kevin) [reply src/components/app.js:4]: Yeah. Style something so we can know that it is working.

Comment (iyaremye-faustin) [reply src/components/app.js:4]: ok let me do it

**Review by SergeNm — CHANGES_REQUESTED** [review]
delete the build folder, correct eslint error & rebase

**Comment by iyaremye-faustin** [comment]
@SergeNm it is already rebased

**Review by rukundo-kevin — DISMISSED** [review]

**Review by NNesta — DISMISSED** [review]
Good job

**Review by SergeNm — APPROVED** [review]

**Review by NNesta — APPROVED** [review]
Good job

**Review by EmyRukundo — APPROVED** [review]

**Pull request merged**